app/assets/javascripts/components/articles/available_article.jsx

```diff
@@ -58,6 +58,7 @@ export const AvailableArticle = createReactClass({
     const ratingClass = `rating ${assignment.article_rating}`;
     const ratingMobileClass = `${ratingClass} tablet-only`;
     const articleLink = <a onClick={this.stop} href={article.url} target="_blank" className="inline">{article.formatted_title}</a>;
+    const isArticle = this.props.course.home_wiki.project !== 'wikidata';
```

Comment:
Similarly to the Revision component, this can be done on a per-article basis with `assignment.project`, and should probably be done with `isWikipedia` rather than `isArticle`.